algorithms/src/main/kotlin/com/kotlinground/algorithms/slidingwindow/repeateddnasequences/findRepeatedDnaSequences.kt

```
@@ -0,0 +1,78 @@
+package com.kotlinground.algorithms.slidingwindow.repeateddnasequences
+
+fun findRepeatedDnaSequences(dnaSequence: String): List<String> {
+    val toIntMap = mapOf(Pair("A", 0), Pair("C", 1), Pair("G", 2), Pair("T", 3))
+    val encodedSequence = arrayListOf<Int>()
+    for (c in dnaSequence) {
+        val n = toIntMap[c.toString()] ?: throw IllegalArgumentException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)